src/test/test-seccomp.c

```diff
@@ -216,6 +218,7 @@ static void test_restrict_namespace(void) {
         }
 
         assert_se(wait_for_terminate_and_warn("nsseccomp", pid, true) == EXIT_SUCCESS);
+#endif
```

Comment:
Oh, sorry. I missed the git msg.

> Because of that I figured this is different from the other cases, and not testing this for now is OK

I'm not sure
```
kernel/fork.c:#ifdef CONFIG_CLONE_BACKWARDS
kernel/fork.c:#elif defined(CONFIG_CLONE_BACKWARDS2)
kernel/fork.c:#elif defined(CONFIG_CLONE_BACKWARDS3)
arch/powerpc/Kconfig:   select CLONE_BACKWARDS
arch/x86/Kconfig:       select CLONE_BACKWARDS                  if X86_32
arch/x86/um/Kconfig:    select CLONE_BACKWARDS
arch/arm/Kconfig:       select CLONE_BACKWARDS
arch/score/Kconfig:     select CLONE_BACKWARDS
arch/arc/Kconfig:       select CLONE_BACKWARDS
arch/xtensa/Kconfig:    select CLONE_BACKWARDS
arch/parisc/Kconfig:    select CLONE_BACKWARDS
arch/arm64/Kconfig:     select CLONE_BACKWARDS
arch/mips/Kconfig:      select CLONE_BACKWARDS
arch/Kconfig:config CLONE_BACKWARDS
arch/Kconfig:config CLONE_BACKWARDS2
arch/Kconfig:config CLONE_BACKWARDS3
arch/microblaze/Kconfig:        select CLONE_BACKWARDS3
arch/cris/Kconfig:      select CLONE_BACKWARDS2
arch/s390/Kconfig:      select CLONE_BACKWARDS2
include/linux/syscalls.h:#ifdef CONFIG_CLONE_BACKWARDS
include/linux/syscalls.h:#ifdef CONFIG_CLONE_BACKWARDS3
```